bim2sim/elements/base_elements.py

```diff
@@ -952,3 +953,23 @@ def create_ifc_mapping(elements: Iterable) -> Tuple[
                            "match if predefined type is not provided.\n%s",
                            no_default)
         return mapping, blacklist, default
+
+
+class SerializedElement:
+    def __init__(self, element):
```

Comment:
Please add docstring for class. What is a serialized element?

Reply:
added with [288095b](https://github.com/BIM2SIM/bim2sim/pull/657/commits/288095baec5a259f22531043f650d4f33e02b75e)